### PR TITLE
Add account deletion reason survey form

### PR DESCRIPTION
**Pull request by mariogalic**
## What does this change?

It enables users to tell us the reason why they wish to delete their account.

## What is the value of this and can you measure success?

Knowing the reason will help us improve user service.

## Screenshots

image

Could you please recommend options to provide to users. The current ones are what Userhelp sees as the most common reason.

@sallygoble @andrewfindlay

### CODE REVIEW

**Comment by sallygoble** [comment]
1) Can you change 'Do you mind telling us why you want to delete your account' to 'Please take a moment to tell us why you want to delete your account' - it's less defensive.
2) I'm not sure we should add 'I've been trolled in the comment section' - I think this makes us look like we aren't looking after our commenters properly. I also think it would be covered under 'privacy'. If people want to go into specific details they can use the Other field. Also - it's a bit passive - we are giving them an opportunity to complain without offering to do something for them, which is less than ideal. 
3) I think the order is fine and the top option should be the default. 

Other than that - where does the info go to?

File: identity/app/views/profile/deletion/accountDeletion.scala.html [hunk 1]
```diff
@@ -80,6 +79,24 @@ <h1 class="identity-title">Are you sure you want to delete your account?</h1>
...
+                <p class="identity-section__text">
+                    <p><b>Do you mind telling us why you wish to delete your account?</b></p>
+
+                    @radioField(accountDeletionForm("reason"), List(
+                        "accident"      -> "I have created an account by accident",
+                        "password"      -> "I accidentally entered my password as the username",
+                        "emails"        -> "I want to stop receiving emails",
+                        "trolls"        -> "I have been trolled in the comments section",
+                        "comments"      -> "I no longer want to comment",
+                        "privacy"       -> "I am concerned about my privacy on the Internet",
```
Comment (katebee) [inline identity/app/views/profile/deletion/accountDeletion.scala.html]: It seems the Guardian convention is to lowercase internet:
> the internet

.. although my preference would just be to go for:

> I am concerned about my privacy online

`</bikeshedding>`

**Comment by mario-galic** [comment]
@sallygoble I have changed trolling to _"I have been exposed to abuse in the comments section"_. Is this ok, or should I remove it?

Survey choice is input into account deletion step function. Once we have it there, we can post the result wherever we want.

**Comment by sallygoble** [comment]
@mario-galic I think we should remove it. As I say - people can let us know using 'other' if they are being trolled.

**Comment by mario-galic** [comment]
Comment abuse choice removed.

**Comment by katebee** [comment]
May be a subject for another PR, but is Saved For Later still applicable in the warnings section?

> Deleting your account will delete your Saved Articles links.

https://github.com/guardian/frontend/pull/16322/files#diff-dd3e896befffa98caa0c3a6b8fe3afeeR74

**Comment by andrewfindlay** [comment]
@katebee It is applicable as SfL is still a feature in the apps.

**Comment by katebee** [comment]
@andrewfindlay Ah, good to know - cheers!

**Review by katebee — APPROVED** [review]
👍 LGTM, however, you may want someone with a bit more Scala skills to look it over, just in case I missed something!

**Pull request merged**